src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-751.mdx

```diff
@@ -0,0 +1,16 @@
+---
+subject: Android agent
+releaseDate: '2024-08-01'
+version: 7.5.1
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.5.1.zip'
+---
+
+## Fix
+
+* Tighten the logic around metric clearance to prevent dropping metrics when harvast is unsuccessful
+* Add sessionId attribute as part of the log reporting
```

Comment:
```suggestion
* Tighten the logic around metric clearance to prevent dropping metrics when harvest is unsuccessful
* Add `sessionId` attribute as part of the log reporting
```